lib/safira/store/buy.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Safira.Store.Buy do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Safira.Store.Redeemable
+  alias Safira.Accounts.Attendee
+
+  schema "buys" do
+    belongs_to(:attendee, Attendee, foreign_key: :attendee_id, type: :binary_id)
+    belongs_to(:redeemable, Redeemable, foreign_key: :redeemable_id, type: :binary_id)
```

Comment:
In Redeemable schema, there is no indication that the primary key is `:binary_id`. By default, when nothing is said, ecto assumes the [primary key](https://hexdocs.pm/ecto/Ecto.Schema.html#module-primary-keys) as `:id`.

Reply:
It makes sense, I'll delete it.